src/views/pages/cart.twig

```diff
@@ -194,7 +194,7 @@
                         <h2 class="font-bold text-sm mb-5">{{ trans('pages.cart.summary') }}</h2>
 
                         <div class="flex justify-between text-sm mb-5">
-                            <span class="text-gray-500">{{ trans("pages.cart.items_total_without_tax") }}</span>
+                            <span class="text-gray-500"> {{ cart.tax_amount ? trans("pages.cart.items_total_without_tax") : trans("pages.cart.items_total") }} </span>
```

Comment:
It will be nicer, if it was

```
<span class="text-gray-500"> {{ trans(cart.tax_amount ? "pages.cart.items_total_without_tax" : "pages.cart.items_total") }} </span>
```